tests/json_schema.go

```diff
@@ -60,6 +60,7 @@ func TestJsonSchemaKeywordLimitation(t *testing.T, fieldPaths []string, schema s
 	mapping := []Mapping{
 		{"errors.context", "context"},
 		{"transactions.context", "context"},
+		{"errors.transaction.id", "transaction.id"},
```

Comment:
@roncohen wasn't that comfortable with this, any way to do this better?

Reply:
Making sure i got this correctly: The test checks that everything that get stored as a `keyword` has the length restriction and vice-versa. We use this mapping to map the json schema names to the final elasticssearch document names. If that's correct, i don't have a better way to do it that what you did here.

Reply:
That's how I understand that test as well, thanks for verifying.